src/OmniSharp.MSBuild/SolutionSdkFileUtil.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using MSB = Microsoft.Build;
+
+namespace OmniSharp.MSBuild
+{
+    class SolutionSdkFileUtil
+    {
+        static MSB.Evaluation.Project GetEvaluatedProject(string filePath,
+            IDictionary<string, string> globalProperties = null)
+        {
+            var projectCollection = globalProperties != null
```

Comment:
Why not use the existing `ProjectLoader` to do this?

Reply:
I had overlooked `ProjectLoader`, I will change to use it.